### DISCUSSION TITLE
Welcome to censys-python Discussions!

### DISCUSSION THREAD

**Discussion by thehappydinoa** [censys/censys-python#52] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by minanagehsalalma** [comment]
@thehappydinoa i am trying to run this command using the censys cli 

`censys search --api-id blah --api-secret blah -q (location.country_code:nowhere) AND autonomous_system.description.raw: "someskitcyisp" --index-type ipv4 --format csv -o 1.csv`

but i am getting this error 

`censys: error: unrecognized arguments: AND autonomous_system.description.raw: someskitcyisp`

i copied the query directly from the search bar in the site as the cli usage docs said 

>**-q <query>, --query <query>**

`a string written in Censys Search syntax
`

**Reply by thehappydinoa** [reply]
Hi @minanagehsalalma,

You're super close to getting it working, the query needs quotes so it knows it's a string. Your command should look like this:

```sh
$ censys search -q "location.country_code: nowhere AND autonomous_system.description.raw: \"someskitcyisp\"" --index-type ipv4 --format csv -o 1.csv
```

>  Please note that I removed the `--api-id/secret` in favor of using the `censys config` command. Check out [our docs](https://censys-python.readthedocs.io/en/latest/quick-start.html) for more info.

**Reply by minanagehsalalma** [reply]
@thehappydinoa oh thanks it worked but how can i output the IPs only ?

**Reply by thehappydinoa** [reply]
@minanagehsalalma 

You need to add the following flags to the end of your command.

```sh
--overwrite --fields "ip"
```
